e2e/cypress/support/api_commands.js

```diff
@@ -432,6 +432,8 @@ Cypress.Commands.add('apiUpdateConfig', (newSettings = {}) => {
             body: settings,
         });
     });
+
+    cy.apiLogout();
```

Comment:
Do logout (sysadmin) after config update so that the specs using this function will explicitly login the test user.